api/services/itemService.js

```diff
@@ -1,9 +1,54 @@
 const { itemDao } = require("../models");
+const generateAdminNumber = require("../utils/generate")
+
+const getItems = async (params) => {
+  const {
+    limit,
+    offset,
+    category,
+    sorting,
+    authorName,
+    itemName
+  } = params;
+  return await itemDao.getItems(limit, offset, category, sorting, authorName, itemName);
+};
+
+const registerItem = async (userId, imageUrl, params) => {
+  const {
+    categoryName,
+    itemName,
+    authorName,
+    productionYear,
+    width,
+    length,
+    height,
+    weight,
+    materials,
+    description,
+    startingBid,
+    biddingStart,
+    biddingTerm
+  } = params
+
+  const categoryId = await itemDao.findCategoryId(categoryName)
+  const materialsArr = await itemDao.insertMaterials(materials)
+  const adminNumber = await generateAdminNumber.generateAdminNumber()
+  const biddingEnd = await itemDao.calculateBiddingEnd(biddingStart, biddingTerm)
+
+  if (await itemDao.isRegistered(userId) === false) {
+    return false
+  } else {
+    return await itemDao.registerItem(userId, imageUrl, categoryId, itemName, authorName, productionYear, width,
+        length, height, weight, materialsArr, adminNumber, description, startingBid, biddingStart, biddingEnd)
+  }
```

Comment:
```suggestion
  if (!itemDao.isRegistered(userId)) {
    return false
  }
  
  return itemDao.registerItem(userId, imageUrl, categoryId, itemName, authorName, productionYear, width,
        length, height, weight, materialsArr, adminNumber, description, startingBid, biddingStart, biddingEnd)
```
1. model 에서 남겨드린 수정사항에 따르면 isRegisterd 함수는 bool 값을 반환하므로 위와 같이 수정될 수 있겠습니다. 
2. 지금의 if else에서 else는 굳이 필요하지 않습니다. 오히려 가독성을 해치게 됩니다.